cli/compose/loader/loader_multiple_test.go

```diff
@@ -0,0 +1,315 @@
+package loader
```

Comment:
I know loader_test.go is already very long, but I think we need to keep the convention of putting tests into a file that matches the filename of the function being tested.  Otherwise it becomes difficult to find tests and figure out what a test is supposed to be testing. Can we keep this in `loader_test.go`, or maybe put the logging test into `merge_test.go` since that is mostly what it's testing?

Reply:
fair point.